src/agent/agent.c

```diff
@@ -886,8 +889,6 @@ bool agent_start(Agent *agent) {
         sd_bus_error error = SD_BUS_ERROR_NULL;
         _cleanup_sd_bus_message_ sd_bus_message *m = NULL;
 
-        fprintf(stdout, "Starting Agent...\n");
```

Comment:
You don't want to keep this one? (If we do, we may want to specify: `Start hirte agent`)

Reply:
This was one of the very first dev debug prints I added. Not sure if it provides much value. We probably need to use `hirte` a bit more and get a feeling where the logging is lacking and need to adjust it. For now, I'd remove it.

Reply:
if you do `systemctl start hirte`, I think it's nice to have in `journalctl` the entry `Starting hirte-agent`, it helps auditability later on

Reply:
systemd would log the start though.